### DISCUSSION TITLE
Mixing Image Prompt and Inpaint

### DISCUSSION THREAD

**Discussion by DanieleRiccelli** [lllyasviel/Fooocus#2381] [Q&A]
hi, 
I'm trying to insert my brand logo onto a mockup t-shirt using Mixing Image Prompt and Inpaint. I'm doing various tests but I always get a different image than my logo.
I deactivated all the styles and loaded my mockup to (inpaint outpaint default) (selecting the part in which to insert the logo) and my logo as image prompt.
Where am I doing wrong?

**Comment by lgwjames** [comment]
see a lot of people talking about this
a lot say just make your image with a person wearing t shirt and then photoshop etc to add brand
use advanced under control tick as you said you have done mixing image and inpaint, and put your brand in image , but use face swap and mask out the area you want on shirt?
